example/js/api/ICU4XDataProvider.d.ts

```diff
@@ -0,0 +1,21 @@
+
+/**
+
+ * An ICU4X data provider, capable of loading ICU4X data keys from some source.
+
+ * See the {@link https://unicode-org.github.io/icu4x-docs/doc/icu_provider/index.html Rust documentation} for more information.
+ */
+export class ICU4XDataProvider {
+
+  /**
+
```

Comment:
I assume the docs will be cleaned up in your next PR.

Reply:
Yup I have a lot of good changes coming, just didn't want to bloat this one too much.